data-processing/entities/sentences/extractor.py

```diff
@@ -123,6 +210,102 @@ def parse(self, tex_path: str, tex: str) -> Iterator[Sentence]:
             length_so_far_in_plain_text = plaintext_end
             tex_sub = tex[start:end]
             context_tex = tex[start - DEFAULT_CONTEXT_SIZE : end + DEFAULT_CONTEXT_SIZE]
+
+            # extract richer context of tex using '\n'
+            extended_tex_sub = extract_richer_tex(context_tex, tex_sub)
+
+            # detect entity fields using pre-defined regex
+            entity_dict = {}
```

Comment:
nitpick: can another word besides `entity` be used here? Elsewhere in this codebase, entity is used to refer to a clickable object in the paper---like a sentence, a citation, a symbol, etc.

Reply:
That's a valid point! How about `tex_unit`? Let me know if you have a better idea!

Reply:
sure i love it!